bot.py

```diff
@@ -1249,7 +1249,17 @@ async def auto_link(ctx):
         content=message_content,
         ephemeral=True
     )
-
+@bot.translate_command(
+    description = "Automatically translate any langugage into another using Google Tranlsate API", aliases = ['tr']
+)
```

Comment:
I can't find `@bot.translate_command` in the list of valid command decorators. To see the valid decorators, see [here](https://docs.pycord.dev/en/master/api.html#bot). Did you mean `@bot.slash_command`?

Additionally, this command would be best as a [Message Context Command](https://docs.pycord.dev/en/master/api.html#messagecommand). It would be awesome if you could implement it this way instead. You could even add a new section of the file like `-------- MESSAGE CONTEXT COMMANDS --------` to keep it organized if you want, too.